lib/contact_list_page.dart

```diff
@@ -4,10 +4,10 @@ import 'package:flutter/services.dart';
 
 import 'add_contact_page.dart';
 import 'chat_page.dart';
-import 'contact.dart';
 import 'profile.dart';
 import 'settings.dart';
 import 'strings.dart';
+import 'db/database.dart';
```

Comment:
Is there some convention of sorting "/" paths after others?

Reply:
No idea. I'll alphabetize them for now and we'll just go with whatever https://dart-lang.github.io/linter/lints/directives_ordering.html wants when we enable it. It sounds like they should just be alphabetized, but automatic enforcement is good.